src/pages/organism/navBar/molecules/userLocation/helper/constData.js

```diff
@@ -0,0 +1,24 @@
+const LOCATIONS = [
```

Comment:
This is not a helper file, this should be in a constant file where this is being used.